de.dlr.sc.virsat.team.ui/src/de/dlr/sc/virsat/team/ui/handler/git/GitCommitHandler.java

```diff
@@ -27,34 +27,35 @@
 /**

```

Comment:
Maybe you can reset this file as you didn't do any changes to it?

Reply:
I did it :) .